tests/vlan/test_vlan.py

```diff
@@ -345,3 +345,111 @@ def test_vlan_tc3_send_invalid_vid(ptfadapter, vlan_ports_list):
         testutils.send(ptfadapter, src_port, invalid_tagged_pkt)
         logger.info("Check on " + str(dst_ports) + "...")
         testutils.verify_no_packet_any(ptfadapter, masked_invalid_tagged_pkt, dst_ports)
+
+@pytest.mark.bsl
```

Comment:
What does this marker do?

Reply:
I chose to include this marker to keep consistency with the three test cases above.
Do you see some issue with this?